### DISCUSSION TITLE
NonceGeekDAO 分享提交指南

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#91] [Announcements]
NonceGeekDAO 鼓励「针对问题的分享」。

好的问题有时候甚至比好的答案更重要。因此，鼓励分享提交申请以问题为导向。

例如：

> 如果实体卡牌游戏区块链化，会是长什么样的？

【分享申请提交方式】

在 [Workshop](https://github.com/WeLightProject/Web3-dApp-Camp/discussions/categories/workshop) 栏目下开贴即可！

帖子的形式如下：

> Title: 你要回答的问题
>
> Content:
> 时间：
> 分享人昵称：
> 内容大纲：

【分享奖励】

DAO 会根据分享的综合程度而定，给分享人发放分享奖励！

**Comment by TianYiFu** [comment]
Title: 游戏化社群运营

Content:
时间：2022.06.23 21:00~22:00(Thur.)
分享人昵称：田一夫（Alfred）
内容大纲：
·游戏化的定义
·游戏化的目的
·游戏化思维与传统思维的差异
·为什么我的“游戏”（社群）不好玩
·游戏化实战——八角行为分析法的应用
·游戏化运营实战案例分享
